styles/login.css

```diff
@@ -0,0 +1,24 @@
+body{
+    background-color: rgb(217, 243, 166);
```

Comment:
Try to use variables for color, rest everything is good

Reply:
sure. i will add the variables files